cmd/dashboard/app/dashboard.go

```diff
@@ -45,11 +44,21 @@ func Run(listenAddress string,
 	}
 
 	// create a platform
-	platformInstance, err := factory.CreatePlatform(logger, platformType, nil)
+	platformInstance, inferredPlatformType, err := factory.CreatePlatform(logger, platformType, nil)
 	if err != nil {
 		return errors.Wrap(err, "Failed to create platform")
 	}
 
+	// set external ip addresses based on platform type
```

Comment:
Would be better to add a platform interface: `GetDefaultInvokeIPAddress()` for two reasons:
1. It's odd that CreatePlatform returns a variable that could easily be evaluated as something like `platformInstance->GetType()`
2. Switching by type should be minimized - that's why interfaces exist